editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -397,9 +407,17 @@ const NewCanvasControlsInner = (props: NewCanvasControlsInnerProps) => {
       onMouseDown={onMouseDown}
       onMouseMove={onMouseMove}
     >
+      <AnimatedPlaceholderBoxes />
```

Comment:
Should this be behind a `when(isFeatureEnabled('Canvas Strategies')` ?

Reply:
fixed!